flash_all.sh

```diff
@@ -4,13 +4,18 @@ set -e
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 cd $DIR
 
+EDL_DIR=$DIR/edl
```

Comment:
let's remove this from the flashing scripts. just like fastboot, you can assume the user ran the setup script